lib/mix/tasks/docs.ex

```diff
@@ -54,6 +54,14 @@ defmodule Mix.Tasks.Docs do
       directory in the output path. Its entries may be referenced in your docs
       under "assets/ASSET.EXTENSION"; defaults to no assets directory.
 
+    * `:before_closing_body_tag` - Literal HTML to be included just before the closing body tag (`</body>`)
+      Useful to inject custom assets, such as Javascript. 
+      Only works with the HTML formatter.
```

Comment:
Because I only added this functionality to the HTML templates :) If you want, I can add it to the EPUB ones.